src/QuantumToolbox.jl

```diff
@@ -3,17 +3,28 @@ module QuantumToolbox
 # Re-export:
 #   1. StaticArraysCore.SVector for the type of dims
 #   2. basic functions in LinearAlgebra and SparseArrays
+#   3. some functions in SciMLOperators
 import Reexport: @reexport
 @reexport import StaticArraysCore: SVector
 @reexport using LinearAlgebra
 @reexport using SparseArrays
+@reexport import SciMLOperators:
+    AbstractSciMLOperator,
+    MatrixOperator,
```

Comment:
Do we really need to re-export all these functions? maybe just `isconstant` is fine. People using them would probably require to do advanced stuff, where importing `SciMLOperators` would be better.

Reply:
maybe
`isconstant`, `iscached`, and `cache_operator`

Reply:
Done